extra/license-overview-generator

```diff
@@ -1508,7 +1508,7 @@ def do_repos():
             if any([file.endswith(".go") for file in files]):
                 do_go_repo(repopath)
                 break
-            elif "gulpfile.js" in files:
+            elif os.path.basename(repopath) in ["gui", "gui-enterprise"]:
```

Comment:
Why? Is `gulpfile.js` wrong?

Reply:
yes: https://github.com/mendersoftware/gui/pull/1924